src/buildblock/HUToMuImageProcessor.cxx

```diff
@@ -101,75 +103,91 @@ Succeeded
 HUToMuImageProcessor<TargetT>::
 virtual_set_up(const TargetT& image)
 {
+#ifdef HAVE_JSON
   this->get_record_from_json();
+#endif
   return Succeeded::yes;
 }
 
-template <typename TargetT>
-void
-HUToMuImageProcessor<TargetT>::
-get_record_from_json()
-{
-  if (this->filename.empty())
-    error("HUToMu: no filename set for the slope info");
-  if (this->manufacturer_name.empty())
-    error("HUToMu: no manufacturer set for the slope info");
+#ifdef HAVE_JSON
+  template <typename TargetT>
```

Comment:
does `clang-format` insert these spaces? If not, I'd rather remove them

Reply:
no, this was markus-format :-) will remove!